UKHomeOffice/cerberus-service
# COP-11199 Add test for Remove main traveller row from co-travellers table

## Description
Add test for Remove main traveller row from co-travellers table
https://support.cop.homeoffice.gov.uk/browse/COP-11199

## To Test
npm run cypress:test:local -- --spec cypress/integration/airpax/airpax-tasks-details.spec.js

Should verify Co-traveller details not available for an AirPax task with a single passenger on task details page


## Developer Checklist

\* Required
